libnetdata/socket/socket.h

```diff
@@ -3,7 +3,9 @@
 #ifndef NETDATA_SOCKET_H
 #define NETDATA_SOCKET_H
 
-#include <openssl/ossl_typ.h>
+#if defined(ENABLE_HTTPS) || defined(ENABLE_DBENGINE)
```

Comment:
This include file is not actually used by dbengine so `defined(ENABLE_DBENGINE)` is not actually doing anything

Reply:
I do not ask nothing from this file too, so I will remove it, the necessary files for SSL are inside the libnetdata/security/socket.h .